src/helpers/Image.php

```diff
@@ -10,6 +10,9 @@
 use Craft;
 use craft\errors\ImageException;
 use craft\image\Svg;
+use Imagick;
+use Throwable;
+use TypeError;
```

Comment:
PHPStorm added these for me but let me know if the `\Imagick`, etc. syntax is preferred.

Reply:
Import all the things!